rules/game_process.tex

```diff
@@ -158,13 +158,24 @@ \subsubsection{Indirect Kick - Champions Cup}
     A deliberate play at the ball includes successfully kicking the ball, dribbling the ball or the goalkeeper playing at the ball with its hands.
   \item If an indirect free kick is kicked directly into the opponent's goal, the goal does not count and a \texttt{Goal kick} is awarded.
   \item Note that an own-goal may always be scored without requiring an indirect kick.
-  \item A direct kick is allowed for the duration of the penalty kick, but this rule still applies after this kick. A contact in the penalty kick counts among this rule.
 \end{itemize}
 
+There are two exceptions to this rule:
+
+\begin{itemize}
+  \item A direct kick is allowed for the duration of the penalty kick, but this rule still applies after this kick.
+  \item If a team is awarded a free kick but fails to execute it within the allotted time, the other team may score directly as long as the second team plays the ball first.
```

Comment:
unsure if second team is also "the other team"?

Reply:
Oh, nice ambiguity spotting skills. I was trying to describe the situation without making up names such as "Team A" and "Team B", but I already had a little difficulty.